### PR TITLE
Update limits.

### PR DESCRIPTION
**Pull request by sergeykostov**

### CODE REVIEW

File: controllers/cbcontainersruntime_controller.go [hunk 1]
```diff
@@ -160,7 +160,7 @@ func (r *CBContainersRuntimeReconciler) setDefaults(cbContainersRuntime *operato
 
 	setDefaultImage(&cbContainersRuntime.Spec.SensorSpec.Image, "cbartifactory/runtime-kubernetes-sensor")
 
-	if err := setDefaultResourceRequirements(&cbContainersRuntime.Spec.SensorSpec.Resources, "64Mi", "30m", "256Mi", "200m"); err != nil {
+	if err := setDefaultResourceRequirements(&cbContainersRuntime.Spec.SensorSpec.Resources, "64Mi", "30m", "1024Mi", "200m"); err != nil {
```
Comment (yurado) [inline controllers/cbcontainersruntime_controller.go:163]: Let's give a higher limit for CPU usage as well. I was thinking something like `500m`

**Comment by sergeykostov** [comment]
Done.

**Review by yurado — APPROVED** [review]

**Review by BenRub — CHANGES_REQUESTED** [review]
@sergeykostov Hi Sergey, please update the kubebuilder defaults in the comments on the api type as well.
This will be relevant when we'll use CRD v1

**Review by yurado — APPROVED** [review]

**Comment by sergeykostov** [comment]
Local branch

**Pull request closed without merging**